src/registrar/assets/js/get-gov.js

```diff
@@ -834,3 +834,436 @@ function hideDeletedForms() {
+   * @param {*} sortBy - the sort column option
+   * @param {*} order - the sort order {asc, desc}
+   * @param {*} loaded - control for the scrollToElement functionality
+   */
+  function loadDomains(page, sortBy = currentSortBy, order = currentOrder, loaded = hasLoaded) {
+    //fetch json of page of domains, given page # and sort
+    fetch(`/get-domains-json/?page=${page}&sort_by=${sortBy}&order=${order}`)
+      .then(response => response.json())
+      .then(data => {
+        if (data.error) {
+          console.log('Error in AJAX call: ' + data.error);
+          return;
+        }
+
+        // handle the display of proper messaging in the event that no domains exist in the list
+        if (data.domains.length) {
+          domainsWrapper.classList.remove('display-none');
+          noDomainsWrapper.classList.add('display-none');
+        } else {
+          domainsWrapper.classList.add('display-none');
+          noDomainsWrapper.classList.remove('display-none');
+        }
+
+        // identify the DOM element where the domain list will be inserted into the DOM
+        const domainList = document.querySelector('.dotgov-table__registered-domains tbody');
+        domainList.innerHTML = '';
+
+        data.domains.forEach(domain => {
+          const expirationDate = domain.expiration_date ? new Date(domain.expiration_date) : null;
+          const expirationDateSortValue = expirationDate ? expirationDate.getTime() : '';
+          
+          const row = document.createElement('tr');
+          row.innerHTML = `
+            <th scope="row" role="rowheader" data-label="Domain name">
+              ${domain.name}
+            </th>
+            <td data-sort-value="${expirationDateSortValue}" data-label="Expires">
+              ${expirationDate ? expirationDate.toLocaleDateString() : ''}
+            </td>
+            <td data-label="Status">
+              ${domain.state_display}
+              <svg 
+                class="usa-icon usa-tooltip usa-tooltip--registrar text-middle margin-bottom-05 text-accent-cool no-click-outline-and-cursor-help" 
+                data-position="top"
+                title="${domain.get_state_help_text}"
+                focusable="true"
+                aria-label="Status Information"
+                role="tooltip"
+              >
+                <use aria-hidden="true" xlink:href="/public/img/sprite.svg#info_outline"></use>
+              </svg>
+            </td>
+            <td>
+              <a href="/domain/${domain.id}">
+                <svg class="usa-icon" aria-hidden="true" focusable="false" role="img" width="24">
+                  <use xlink:href="/public/img/sprite.svg#${domain.state === 'deleted' || domain.state === 'on hold' ? 'visibility' : 'settings'}"></use>
+                </svg>
+                ${domain.state === 'deleted' || domain.state === 'on hold' ? 'View' : 'Manage'} <span class="usa-sr-only">${domain.name}</span>
```

Comment:
That's in the JS though, where I don't think we have access to the model props

Reply:
@rachidatecs you're accessing the model property "state" when you do .state, though, you sure doing domain.State won't work? domain should be added to the context which is why you can get state, name, expiration date with the "." operator. I would be suprised if .State didn't work but the others do. Though happy to learn something new here.

Reply:
Way I see it, what I can access in the JS is whatever I set in the json API, ie:

```
# Convert objects to JSON-serializable format
    domains = [
        {
            "id": domain.id,
            "name": domain.name,
            "expiration_date": domain.expiration_date,
            "state": domain.state,
            "state_display": domain.state_display(),
            "get_state_help_text": domain.get_state_help_text(),
            "action_url": f"/domain/{domain.id}",
        }
        for domain in page_obj.object_list
    ]
```

so only .state and only because I set it.

Reply:
Found a workaround